samples/tfm/provisioning_image/README.rst

```diff
@@ -0,0 +1,85 @@
+.. _provisioning_image:
+
+Provisioning image
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to initialize the provisioning process of a Trusted Firmware M(TF-M) image.
+The provioning process sets the device in the Provisioning Platform Root of Trust(PRoT) Security Lifecycle as defined in the security model of Platform Security Architecture(PSA).
+This sample does not include a TF-M image, it is a Zephyr image which is required to run before the TF-M image in order to allow the provisioning of the TF-M image later.
+
+Requirements
+************
+
+The following development kits are supported:
+
+.. table-from-sample-yaml::
+
+The :ref:`lib_hw_unique_key` and the :ref:`lib_identity_key` libraries are required to generate and store the Master Key Encryption Key (MKEK) and the identity key into KMU.
+
+.. note::
+   Once the required identity key is provisioned on the device, only the code pages should be erased as ERASEALL removes the identity key from the system.
```

Comment:
is code pages the term here? to me it means something else.

Reply:
I suppose internal flash would be less ambiguous.

Reply:
Or, "the UICR should not be erased, as that removes the identity key from the system"